scripts/update_robot.py

```diff
@@ -183,7 +184,55 @@ def on_shutdown():
     return nl.rc
 
 
+def rethink_ssh_update_version():
+    """
+    Verifies the version of the software is older than 1.1.0.
+    If newer, print out url to SSH update instructions
+    and return False
+    """
+    param_name = "rethink/software_version"
+    robot_version = rospy.get_param(param_name, None)
+    if not robot_version:
+        rospy.logerr("RobotUpdater: Failed to retrieve robot version "
+                     "from rosparam: %s\n"
+                     "Verify robot state and connectivity "
```

Comment:
After talking with Dihan a bit, it appears the answer is still yes, ROS does need to be running on the robot.